projects/plugins/crm/includes/ZeroBSCRM.AJAX.php

```diff
@@ -1104,8 +1104,7 @@ function ZeroBSCRM_accept_quote() {
 
 		// validate that this has been posted by the contact associated with the quote
 		if ( ! $uinfo->ID
-			|| zeroBS_getCustomerIDWithEmail( $uinfo->user_email ) !== zeroBSCRM_quote_getContactAssigned( $quoteID )
-			|| zeroBSCRM_permsQuotes()
+			|| zeroBS_getCustomerIDWithEmail( $uinfo->user_email ) !== zeroBSCRM_quote_getContactAssigned( $quoteID ) // phpcs:ignore
```

Comment:
I believe technically this second condition could result in `false !== false`?

I don't see any real security risk, but imagine I'm a WP subscriber with no CRM access. I'd be logged in (so have a user ID set), but I have a no email in CRM, so if a quote is not assigned to a contact, in theory I can accept it.

Reply:
But we can add an extra condition (paranoid mode):

`if ( ! $uinfo->ID || zeroBSCRM_quote_getContactAssigned( $quoteID ) === false || zeroBS_getCustomerIDWithEmail( $uinfo->user_email ) !== zeroBSCRM_quote_getContactAssigned( $quoteID )`

Reply:
I'm probably overly paranoid. :^)